robot/basestation/stream_capture.py

```diff
@@ -64,6 +66,9 @@ def stop_recording_feed(stream_url):
     success = True
     try:
         proc_video[stream_url].communicate(b'q')
+        if camera_rotation != '0':
```

Comment:
You should really store the rotation as an integer. Cast it to an int worst case but you must handle exceptions so the server doesnt crash

Reply:
I also suggest you don't do this check but just pass in zero